.github/workflows/ci.yml

```diff
@@ -30,6 +30,9 @@ jobs:
         uses: cashapp/activate-hermit@v1
       - name: Build Cache
         uses: ./.github/actions/build-cache
+      # TODO: remove, testing this
+      - name: Build Kotlin
+        run: just build-kt-runtime
```

Comment:
there's an issue with caching happening; `Test Go` fails without this step because `buildengine` exercises Kotlin code in the Go tests now. the cached build uses a kotlin-runtime version that's missing the necessary local changes from this PR

Reply:
As we discussed, this should stay here.

Reply:
I'm actually surprised the `ftl` development build didn't do this automatically.